ganga/GangaLHCb/Lib/Applications/GaudiExec.py

```diff
@@ -443,6 +443,9 @@ def getEnvScript(self, isLbEnv):
         """
         Return the script which wraps the running command in a correct environment
         """
+        j = self.getJobObject()
+        if j.virtualization:
+            return 'singularity exec -e --bind /cvmfs %s bash -c " source /cvmfs/lhcb.cern.ch/lib/LbEnv && source LbLogin.sh -c %s && ' % (j.virtualization.image, self.platform)
```

Comment:
This assumes that `singluarity` is the virtualisation environment used. Can we make it more generic?